### PR TITLE
LPD-26955 Migrate quarantine test FriendlyURL.testcase

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPD-26955)

### Changes

Migrate not only the quarantine test but FriendlyURL.testcase complete

> [!NOTE]
>Needs backend review because I'm removing tests due existing integration tests, so I would like to have a double check on that.

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/test/playwright/tests/layout-admin-web/pageFriendlyURL.spec.ts [hunk 1]
```diff
@@ -117,3 +117,38 @@ test('Canonical URL doesnt change with localized Friendly URL.', async ({
...
+	// The configuration view should only allow setting the FriendlyURL and the associated language
+
+	await layoutPage.goto(site.friendlyUrlPath);
+	await layoutConfigurationPage.goto('Test Page Name', 'General');
+	await layoutConfigurationPage.setGeneralLayoutConfiguration(
+		'/test-pagina',
+		'spanish'
+	);
+
+	await page.goto('/es/web/' + site.name + '/test-page-name');
```
Comment (sandrodw3) [inline modules/test/playwright/tests/layout-admin-web/pageFriendlyURL.spec.ts:149]: You can also use `layout.friendlyUrlPath` here

File: modules/test/playwright/tests/layout-admin-web/pageFriendlyURL.spec.ts [hunk 2]
```diff
@@ -71,3 +75,45 @@ test('Navigating to the URL of an uncreated portlet does not throw errors.', asy
...
+
+	// Create a new incognito browser context
+
+	const context = await browser.newContext();
+
+	// Create a new page inside context.
+
+	const newPage = await context.newPage();
+
+	await newPage.goto('/es/web/' + site.name + '/test-page-name');
```
Comment (sandrodw3) [inline modules/test/playwright/tests/layout-admin-web/pageFriendlyURL.spec.ts:114]: You can store the response from `apiHelpers.headlessDelivery.createSitePage` in a `layout` variable and here do `await.page.goto(layout.friendlyUrlPath)` 😄

**Review by sandrodw3 — COMMENTED** [review]
@BarbaraCabrera nice work! ✨ It looks super good overall, I just suggested improvements regarding names, file organisation and minor adjustments, let me know what you think please 🥰

**Comment by BarbaraCabrera** [comment]
I've made the changes required (the last two commits) but the tests into siteTemplatesWithWebContent.spec.ts are failing locally, even before the changes were applied. Could you please let me know if this also happens in your local? @sandrodw3 🙏🏼

**Comment by sandrodw3** [comment]
LGTM! 🥳 leaving it on hold until we see ci:test:playwright results

**Comment by sandrodw3** [comment]
@BarbaraCabrera two tests are failing, for the first of them you have to replace localhost:8080 with `liferayConfig.environment.baseUrl`. For the second one, I'm not sure why it's failing since the reports in Testray are not giving much detail right now, but I have some ideas we could try

**Comment by BarbaraCabrera** [comment]
ci:test:playwright

**Comment by BarbaraCabrera** [comment]
ci:test:playwright

**Comment by BarbaraCabrera** [comment]
ci:test:playwright

**Comment by BarbaraCabrera** [comment]
ci:test:playwright

**Comment by sandrodw3** [comment]
ci:forward

**Comment by sandrodw3** [comment]
ci:test:sf

**Comment by sandrodw3** [comment]
https://github.com/liferay-echo/liferay-portal/pull/15795

**Pull request closed without merging**